shell/AIShell.Kernel/Command/McpCommand.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.CommandLine;
+using AIShell.Abstraction;
+
+namespace AIShell.Kernel.Commands;
+
+internal sealed class McpCommand : CommandBase
+{
+    public McpCommand()
+        : base("mcp", "Command for managing MCP servers and tools.")
+    {
+        this.SetHandler(ShowMCPData);
+
+        //var start = new Command("start", "Start an MCP server.");
```

Comment:
Should these commends be removed?

Reply:
The code was left intentionally. The code was for adding `/mcp start` and `/mcp stop`, which are follow-up work listed in the PR description.